pkg/engine/params.go

```diff
@@ -252,5 +252,5 @@ func getParameters(cs *api.ContainerService, generatorCode string, aksEngineVers
 		}
 	}
 
-	return parametersMap, nil
+	return parametersMap
```

Comment:
This only returned `nil` for error, so that was removed from the function signature.